app/controllers/admins/questions_controller.rb

```diff
@@ -0,0 +1,57 @@
+class Admins::QuestionsController < ApplicationController
+  before_action :require_admin_logged_in
+
+  def index
+    questions = if !!params[:words]
```

Comment:
```suggestion
if params[:words].present?
```

個人的には可読性の観点と、あと空文字の場合に無駄なクエリが走らないので、こっちの方がよいかなと思いましたが、どうでしょう？　逆に`!!`のメリットがあるようであれば、教えていただけると勉強になるのでありがたいです！

Reply:
!!つけるとその分クエリが走るんですね！
逆の逆だからってことですか？
present?だと一発だからいいということか～！私もそうしときます。
!!のメリットは、なんでしょう…短いということでしょうか？？笑

Reply:
> !!つけるとその分クエリが走るんですね！

いや、こういう事実がまずあるんです！

```rb
irb(main):005:0> !''
=> false
irb(main):006:0> !nil
=> true
```

なので、空文字がある場合、LIKE検索がされてしまうので、微妙に遅くなってしまうんじゃないかなと。
ただ、実際に発行されるクエリを確認して発言しているわけではないから、間違っていたらごめんなさい😓 

> !!のメリットは、なんでしょう…短いということでしょうか？？笑

短いの好き勢ですか笑
私は、長くても構わん勢なので、ちょっと宗教が違うかもしれないですね笑

強制はしないですし、私はある程度英語ができるという背景があってのことかもしれないですが、`present?`と訊いてくれると直感的な感じがして好きなんですけど、空文字云々の話を除けば、趣味みたいなものに近いかもしれないので、こういう人もいるんだなと思っておいてください！

Reply:
なるほど、そうなのか。
空文字の場合というのを考えられてなかったです。
だったらpresent?の方がいい気がしますね😊
!!とpresent?の違いを詳しく理解していなかったので、だったら短い方がいいじゃんって思っていたんですけど、
そのように考えたらpresent?の方がいいですね！